### DISCUSSION TITLE
Phi-3.5-vision instruct playground - How to put an image?

### DISCUSSION THREAD

**Discussion by ai-bits** [community/community#144475] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I open a Playground from the Models Marketplace there is nothing to put an image.
VS Code Playground from the Models Catalog shows a grayed-out Attachment paper clip.
Phi returns nonsense for a pasted image of a protein structure.
![protein-structure-dark](https://github.com/user-attachments/assets/7e13d299-88c1-4355-9d76-3d3a1bd57e91)

Regards
G.

**Comment by mcasalaina** [comment]
I just tried this and it seems to be working fine, I was able to attach an image.

**Reply by ai-bits** [reply]
If there is no Attachment paper clip or it is greyed out and paste does not seem to work, do you put the path into the prompt text? (Didn't try the latter.)
Thx
G.

**Comment by mcasalaina** [comment]
There is an attachment paper clip button, and it works as expected.

**Comment by ai-bits** [comment]
Great NON-ANSWER again!

**Comment by KateCatlin** [comment]
Hey @ai-bits! Thanks for letting us know. Is this still an issue for you or has it since resolved?

**Comment by ai-bits** [comment]
THREE MONTHS LATER (MORE LIKE 3 YEARS IN AI) you ask!?
